pages/home/use-tools/call-tools-directly.mdx

```diff
@@ -4,64 +4,108 @@ description: "Learn how to call tools programmatically using Arcade"
 ---
 
 import { Steps, Tabs } from "nextra/components";
+import { SignupLink } from "@/components/Analytics";
 
-# Call tools directly
+# Call Tools Directly
 
-In this guide, you'll learn how to call tools programmatically using Arcade.
+In this guide, you'll learn how to call tools directly, instead of having the [LLM choosing](/home/use-tools/call-tools-with-models) which tool to call.
 
-Unlike [calling tools with models](/home/use-tools/call-tools-with-models), calling tools directly doesn't use an AI model to decide which tool to call and what arguments to pass. Instead, you specify the tool and its arguments in code.
-
-You still get the benefits of Arcade managing authorization and tool execution, but you have more control over when and how the tool is called. This is useful in apps and AI agents where the workflows are more complex than a single model call.
+Direct tool calling is useful in situations where you already know what tool is needed or when there is no prompt for an LLM to interpret and decide on the tool calling. It also gives you full control over what is executed by the Arcade Engine and how.
 
 <Steps>
 
 ### Prerequisites
 
-- [Set up Arcade](/home/quickstart)
-- Create an Arcade [API key](/home/api-keys), if you haven't already
+1. Create an <SignupLink utmParams={{
+            utm_source: "docs",
+            utm_medium: "page",
+            utm_campaign: "quickstart",
```

Comment:
this isn't the quickstart page so we should name this something different

Reply:
fixed